packages/api-v2/src/plan/entities/plan.entity.ts

```diff
@@ -28,9 +28,6 @@ export class Plan {
   @Column()
   major: string;
 
-  @Column()
-  coopCycle: string;
-
```

Comment:
we don't need this field anymore cause it is quite useless, there are a bunch of changes regarding this